packages/core/src/utils/storage/index.ts

```diff
@@ -10,6 +11,12 @@ export const buildUploadFile = (config: StorageProviderData): UploadFile => {
 
     return storage.uploadFile;
   }
+  if (config.provider === 'GoogleStorage') {
+    const { projectId, keyFilename, bucketName } = config;
+    const storage = buildGoogleStorage(projectId, keyFilename, bucketName);
+
+    return storage.uploadFile;
+  }
```

Comment:
Would also be great if you can add a test case for L14-L19. Thanks

Reply:
Where should i add the test case ?

Reply:
We don't have tests for storage providers, so it is OK, I'll add tests later.

Reply:
OK, no worries